test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import { v4 as uuid } from 'uuid';
+
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+async function getTestImage() {
+	const testImagePath = path.join(
+		__dirname,
+		'..',
+		'..',
+		'..',
+		'assets',
+		'10x10_e2e_test_image_z9T8jK.png'
+	);
+	const filename = uuid();
+	const tmpFilename = path.join( os.tmpdir(), filename + '.png' );
+	fs.copyFileSync( testImagePath, tmpFilename );
+	return tmpFilename;
+}
+
+async function getBackgroundColorAndOpacity( locator ) {
+	return await locator.evaluate( ( el ) => {
+		const computedStyle = window.getComputedStyle( el );
+		return [ computedStyle.backgroundColor, computedStyle.opacity ];
+	} );
+}
+
+test.describe( 'Cover', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can set overlay color using color picker on block placeholder', async ( {
+		page,
+		editor,
+	} ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+
+		// Locate the Black color swatch.
+		const blackColorSwatch = page.getByRole( 'button', {
+			name: 'Color: Black',
+		} );
+		await expect( blackColorSwatch ).toBeVisible();
+
+		// Get the RGB value of Black.
+		const blackRGB = await blackColorSwatch.evaluate(
+			( node ) => node.style.backgroundColor
```

Comment:
Instead of relying on the "style" attribute, which is an implementation detail, maybe we should use `window.getComputedStyle` here to get the computed color instead.